### PR TITLE
🙏  add GDPR delete to the FAQ

### PR DESCRIPTION
**Pull request by zoltan-david**
### Description

By now, we received a few questions from customers about the GDPR delete flow.
In this commit, the answers to their most basic questions are added to the FAQ.

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

File: website/docs/faq.md [hunk 1]
```diff
@@ -132,6 +132,16 @@ Yes! You can export and download your current product as a standard JSON file an
 - All tags
 - All segments
 
+### How long do you keep my data?
```
Comment (sigewuzhere) [inline website/docs/faq.md:135]:
```suggestion
### How long does ConfigCat keep my data?
```

File: website/docs/faq.md [hunk 2]
```diff
@@ -132,6 +132,16 @@ Yes! You can export and download your current product as a standard JSON file an
...
+
+We keep organization data as long as we see activity in that organization.
+After several months of inactivity, we send you a series of email notifications about or plans to delete your organization and all associated data.
+We consider an organization inactive if it meets all of the following criteria:
+- no audit log events are generated in the organization,
+- no valid REST API calls are made to the organization,
```
Comment (sigewuzhere) [inline website/docs/faq.md:141]: ez melyik API? Rest apirol mashol nem irunk. van dashboard api es public mgmt api.

File: website/docs/faq.md [hunk 3]
```diff
@@ -132,6 +132,16 @@ Yes! You can export and download your current product as a standard JSON file an
...
+- none of the organization's config JSONs are downloaded,
```
Comment (sigewuzhere) [inline website/docs/faq.md:142]:
```suggestion
- config JSONs aren't downloaded from the ConfigCat CDN,
```

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

File: website/docs/faq.md [hunk 4]
```diff
@@ -132,6 +132,16 @@ Yes! You can export and download your current product as a standard JSON file an
 - All tags
 - All segments
 
+### How long does ConfigCat keep my data?
+
+We keep organization data as long as we see activity in that organization.
+After several months of inactivity, we send you a series of email notifications about or plans to delete your organization and all associated data.
+We consider an organization inactive if it meets all of the following criteria:
+- no audit log events are generated in the organization,
+- no valid REST API calls are made to the organization,
```
Comment (zoltan-david) [inline website/docs/faq.md:141]:
```suggestion
- no valid calls are made to the organization via the Public Management API,
```

File: website/docs/faq.md (hunk 2 above)
Comment (zoltan-david) [reply website/docs/faq.md]: beírtam most, hogy Public Management API

**Review by sigewuzhere — APPROVED** [review]

**Pull request merged**